pkgs/by-name/il/illuminanced/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  nix-update-script,
+}:
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "illuminanced";
+  version = "1.0.2";
+
+  src = fetchFromGitHub {
+    owner = "mikhail-m1";
+    repo = "illuminanced";
+    tag = "${finalAttrs.version}";
+    sha256 = "sha256-ZEVma0uj9rsWB+vfUL7w3dHxI/ppBCG23TirGE+RREk=";
```

```suggestion
    hash = "sha256-ZEVma0uj9rsWB+vfUL7w3dHxI/ppBCG23TirGE+RREk=";
```

I remembered these were suggested by someone???

---

done